### ISSUE TITLE
t.infoemails.microsoft.com

### ISSUE THREAD

**Issue by 4d62** [hagezi/dns-blocklists#4518]
### Which domain(s) should be blocked?

t.infoemails.microsoft.com

### Why should these domain(s) be blocked?

Microsoft Azure tracking pixel.

Attn: Breaks the "open in webpage" in emails (Link `https://t.infoemails.microsoft.com/r/?id=[...]&e=[...]&s=[...]`). The email is viewed in `m.infoemails.microsoft.com` (Link `https://m.infoemails.microsoft.com/rest/head/mirrorPage/[...].html`) so that isn't broken. The email tracking pixel is loaded as `https://t.infoemails.microsoft.com/r/?id=[...]` at the bottom of the email tho.

Basically: blocking the domain will block the tracking pixel for users who view the email in their inbox, but breaks for users which attempt to open the email in a web browser.

### I confirm ...

- [X] that I have checked if there is no other issue for the domain(s) whereby they were unblocked or the blocking was declined.
- [X] that I have checked that the domain(s) are not already blocked.

### Privacy

- [X] I confirm that the report does not contain any private information.
